Common/src/CConfig.cpp

```diff
@@ -2862,34 +2860,61 @@ void CConfig::SetConfig_Parsing(istream& config_buffer){
           if (!option_name.compare("RELAXATION_FACTOR_ADJFLOW"))
             newString.append("Option RELAXATION_FACTOR_ADJFLOW is now RELAXATION_FACTOR_ADJOINT, "
                              "and it also applies to discrete adjoint problems.\n\n");
-          if (!option_name.compare("WRT_MESH_QUALITY"))
+          else if (!option_name.compare("WRT_MESH_QUALITY"))
```

Comment:
Wouldn't this prevent multiple warnings from being printed? Not sure if that's the point (or Im misunderstanding).

Reply:
No, those checks for deprecated options were already mutually exclusive.
You get multiple messages from the outer loop over all options in the config, and so you will also get multiple "did you mean?"s if you misspell more than one option.

Reply:
Ahh I missed the outer loop. Thanks.